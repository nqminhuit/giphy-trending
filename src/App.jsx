import React from "react";
import "./App.scss";
import AppRoutes from "./components/AppRoutes.jsx";
import ErrorBoundary from "./view/ErrorBoundary.jsx";

export default function App() {
  return (
    <ErrorBoundary>
      <AppRoutes />
    </ErrorBoundary>
  );
}
