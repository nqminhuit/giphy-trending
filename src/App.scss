// @import "~bootstrap/scss/functions";
// @import "~bootstrap/scss/variables";
// @import "~bootstrap/scss/mixins";
// @import "~bootstrap/scss/utilities";
// @import "~bootstrap/scss/containers";
// @import "~bootstrap/scss/grid";

// @import "~bootstrap/scss/utilities/api";

@import "~bootstrap/scss/bootstrap";

@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap');

body {
  font-family: "Roboto";
  font-weight: 400;
}

.h-200p {
  height: 200px;
}

.cl-royalblue {
  color: royalblue
}
