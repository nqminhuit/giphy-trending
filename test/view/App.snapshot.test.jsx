import React from "react";
import renderer from "react-test-renderer";
import App from "../../src/App.jsx";

it("should render <App />", () => {
  const component = renderer.create(<App />).toJSON();
  expect(component).toMatchSnapshot();
});
