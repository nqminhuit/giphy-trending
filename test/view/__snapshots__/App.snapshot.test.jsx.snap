// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <App /> 1`] = `
<div
  className="container"
>
  <h1
    className="text-center"
  >
    Wellcome to Giphy Trending!
  </h1>
  <div
    className="bg-light d-flex flex-wrap justify-content-evenly"
  />
</div>
`;
